embuilder.py

```diff
@@ -127,6 +127,8 @@ def main():
                       help='build relocatable objects for suitable for dynamic linking')
   parser.add_argument('--force', action='store_true',
                       help='force rebuild of target (by removing it first)')
+  parser.add_argument('--wasm64', action='store_true',
+                      help='use wasm64 architecture')
```

Comment:
FYI `tools/gen_struct_info.py` will need the exact same command line argument  I think

Reply:
ok